app/client/index.js

```diff
@@ -1,5 +1,17 @@
 import { logDebug, logError } from '../util/logger'
 import contractFn from './contractFn'
+import TokenPriceDataFeed from './TokenPriceDataFeed'
+
+export const getDataFeedLatestResult = async ({ address }) => {
```

Comment:
might want to change the param name to `dataFeedAddress` so it's more obvious what it is